### PR TITLE
Html mainpage

### PR DESCRIPTION
**Pull request by julialoeschel**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="24"
+          height="24"
+          viewBox="0 0 24 24"
+        >
+          <path d="M18 24l-6-5.269-6 5.269v-24h12v24z" />
+        </svg>
+      </li>
+      <li>
```
Comment (F-Kirchhoff) [inline index.html:93]: If these are links you can use a achor tag.

**Comment by F-Kirchhoff** [comment]
nice! u have to show me how to use svgs in html sometime :)

**Review by F-Kirchhoff — COMMENTED** [review]
maybe use extra achor tags in your navbar.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles/index.css" />
+  </head>
+  <body></body>
+</html>
+<body>
+  <h1>Quiz-App</h1>
+
+  <div>
```
Comment (badabam) [inline index.html:15]: I'd go with a `section` here.

**Pull request merged**